robottelo/datafactory.py

```diff
@@ -277,17 +317,14 @@ def valid_org_names_list():
 
 @datacheck
 def valid_usernames_list():
-    """Returns a list of valid user names.
-
-    Note: utf8 data is being rejected by the satellite server so it is not
-    included
-
-    """
+    """Returns a list of valid user names."""
     return [
         gen_string("latin1"),
         gen_string("alpha"),
         gen_string("alphanumeric"),
         gen_string("numeric"),
+        gen_string("utf8"),
+        gen_string('cjk'),
```

Comment:
@elyezer it depends. in most cases it counts the number of bytes, but sometimes it counts the number of chars (example - hostgroups), so imo the best way is to hardcode boundaries and check if it works :)
@rochacbruno for utf8 it may be either `gen_string('utf8', random.randint(1, 100)),` or somewhere at `gen_string('utf8', random.randint(1, 33)),`, depending on whether sat validates number of bytes or chars. If it's bytes number - i'm not sure about exact value, 33 is just a guess (it's just usually 1/3 of chars length), but it's not that important as we don't test boundary for such case. Just try to hardcode different boundaries and see if that works.
Regarding other string types - it should be `gen_string('***', random.randint(1, 100)),`, as satellite should accept any length in that range (and btw we had some weird bugs before with some entities not accepting names of 1 chars, so it's worth covering such case).